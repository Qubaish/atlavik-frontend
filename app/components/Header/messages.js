/*
 * HomePage Messages
 *
 * This contains all the text for the HomePage component.
 */
import { defineMessages } from 'react-intl';

export const scope = 'components.Header';

export default defineMessages({
  hotels: {
    id: `${scope}.hotels`,
    defaultMessage: 'Hotels',
  },
});
