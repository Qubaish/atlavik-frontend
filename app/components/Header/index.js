import React from 'react';
import { Link } from 'react-router-dom';
import { FormattedMessage } from 'react-intl';
import messages from './messages';

function Header() {
    return (
        <nav className="grey darken-3">
            <div className="nav-wrapper">
                <a href="#" className="brand-logo right">
                Logo
                </a>
                <ul id="nav-mobile" className="left hide-on-med-and-down">
                <li>
                    <Link to="/">
                    <FormattedMessage {...messages.hotels} />
                    </Link>
                </li>
                </ul>
            </div>
        </nav>
    )
}

export default Header;
