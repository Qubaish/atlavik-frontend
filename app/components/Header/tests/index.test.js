import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Header from '../index';

describe('<Header />', () => {
    it('should render a DropDown', () => {
        const container = shallow(
            <Header />
        );
        expect(shallowToJson(container)).toMatchSnapshot();
    });
});
