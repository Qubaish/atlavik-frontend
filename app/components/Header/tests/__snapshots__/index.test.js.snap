// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render a DropDown 1`] = `
<nav
  className="grey darken-3"
>
  <div
    className="nav-wrapper"
  >
    <a
      className="brand-logo right"
      href="#"
    >
      Logo
    </a>
    <ul
      className="left hide-on-med-and-down"
      id="nav-mobile"
    >
      <li>
        <Link
          to="/"
        >
          <FormattedMessage
            defaultMessage="Hotels"
            id="components.Header.hotels"
            values={Object {}}
          />
        </Link>
      </li>
    </ul>
  </div>
</nav>
`;
